osf/models/registrations.py

```diff
@@ -905,7 +912,14 @@ def system_tags(self):
     @classmethod
     def create_from_node(cls, user, schema, node=None, data=None, provider=None):
         if not provider:
-            provider = RegistrationProvider.load('osf')
+            provider = RegistrationProvider.get_default()
+
+        if provider.is_default:
+            # If the default provider doesn't have schemas specified yet, allow all schemas
```

Comment:
We should just migrate the schemas in there, it's actually a many-to-many relationship now with schemas and providers so it won't be too difficult.

Reply:
Are you saying that the logic of the if-block should be to add the schemas to the default provider? Do you happen to know what specific registrations should be allowed/added to the provider here/is it better to create a hard-coded list of schemas that is added here as opposed to allowing anything until an admin specifies what should be allowed in the admin app?

Reply:
Yeah I don't actually see why this if block is necessary at all. Even a migration shouldn't be necessary because admins can select which schemas they want once this is merged.

Reply:
The idea is that there may be a brief amount of time between the release and an admin associating schemas to the default provider where no one will be able to create draft registrations. And that handling it with this block makes things a bit easier for staging/testing environments.